### PR TITLE
Dedicated Mouse over Legen label

### PR DESCRIPTION
**Pull request by SidSyrus**
- Fixed bug with npm dependency on loadsh 'uniq' function
- New OncoGrid Param for track legend label
- Added track legend label to OncoGrid graph
- Moved track group rendering from group label to legend label

### CODE REVIEW

**Comment by btiernay** [comment]
Screen shot?

**Comment by SidSyrus** [comment]
image

File: src/TrackGroup.js [hunk 1]
```diff
@@ -143,6 +146,15 @@ OncoTrackGroup.prototype.init = function (container) {
...
         .attr('class', _self.prefix + 'track-group-label')
         .text(_self.name);
+    
+    _self.legend = _self.container.append('text')
+        .attr('x', _self.width)
+        .attr('y', -7)
+        .attr('dy', '.32em')
+        .attr('text-anchor', 'end')
+        .attr('class', _self.prefix + 'legend-group-label')
+        .style('fill', 'blue')
```
Comment (andricDu) [inline src/TrackGroup.js]: We should probably do this in a style rather than hardcode a default colour.

Comment (SidSyrus) [reply src/TrackGroup.js]: Sure. So I think that styling would have to be done from Portal side though, right?

Comment (andricDu) [reply src/TrackGroup.js]: Yup! https://github.com/icgc-dcc/dcc-portal/blob/develop/dcc-portal-ui/app/scripts/oncogrid/styles/_oncogrid.scss#L216

Just provide a simple example in the test page.

**Pull request merged**